pages/api/generate-token.ts

```diff
@@ -50,11 +51,14 @@ const onfido = getOnfido();
 export default async function handler(req: NextApiRequest, res: NextApiResponse<ApplicantTokenPair | unknown>) {
   try {
     const ipAddress = req.headers['x-real-ip'] ?? req.connection.remoteAddress;
+    const countryOfResidence = getCountryCodeFromTimezone(req.body.timezone);
+    console.log('countryOfResidence', countryOfResidence);
```

Comment:
Are console logs printed in prod? If yes, console logs should be removed or disabled in prod.

Reply:
This code is run by the backend so it's a console log displayed in the backend for logging purpose.

Reply:
It's not printed to the browser.